### ISSUE TITLE
Travis builds fail under Dart 1.24-dev

### ISSUE THREAD

**Issue by chalin** [dart-lang/site-webdev#685]
E.g., see this Travis log: https://travis-ci.org/dart-lang/site-webdev/jobs/239179187
```nocode
$ content_shell --dump-render-tree
[7723:7723:0603/225244:194241839:ERROR:browser_main_loop.cc(219)] Running without the SUID sandbox! See https://chromium.googlesource.com/chromium/src/+/master/docs/linux_suid_sandbox_development.md for more information on developing with the sandbox on.
Xlib:  extension "RANDR" missing on display ":99.0".
[7740:7740:0603/225244:194312547:ERROR:sandbox_linux.cc(334)] InitializeSandbox() called with multiple threads in process gpu-process
```
The output above is normal, **except** for the last line:
```nocode
[7740:7740:0603/225244:194312547:ERROR:sandbox_linux.cc(334)] InitializeSandbox() called with multiple threads in process gpu-process
```

@kevmoo - is content_shell expected to still run under 1.24?

cc @kwalrath

**Comment by chalin** [comment]
Trying out @keertip's suggestion:
> The flag `--dump-render-tree` is no longer valid. ... 
change it to `--run-layout-test` 
and [possibly] `--disable-gpu-early-init` and `--disable-gpu`

**Comment by chalin** [comment]
The new command line flags did the trick for `content_shell`.

And with the latest release of the `test` package (0.12.21), all Travis builds are green once again. \o/

**Comment by chalin** [comment]
Fixed by #695 and #702.
